### PR TITLE
Documentation subpage in frame

### PR DESCRIPTION
**Pull request by thenor**

### CODE REVIEW

**Comment by thenor** [comment]
**Note:** The reason for the incomplete label is that I have yet to try this with the alabaster theme for SWF.

File: documentation.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 layout: page
-title: Docs
-external-link: //pelux.io/software-factory
+title: Documentation
```
Comment (JEderonn) [inline documentation.md]: I would propose not rename Docs, or to use the term "Handbook" instead. Perhaps check with Andreas? Note that we will add a section for Architecture soon and then we likely need to change the term again.

Comment (thenor) [reply documentation.md]: I agree with calling it Handbook instead. I have felt that "Documentation" has felt too general.

**Comment by thenor** [comment]
I updated the link from resources to point to the Handbook page instead. The idea for me is that this page is for the handbook connected to the latest release. The releases page links for main releases are kept as they were, since they point to specific versions of the Pelux Develepment Handbook.

**Pull request merged**